.github/DEVELOPMENT.md

```diff
@@ -19,28 +19,29 @@ iOS and MacCatalyst will require current stable Xcode. You can get this [here](h
 
 If you're missing any of the Android SDKs, Visual Studio should prompt you to install them. If it doesn't prompt you then use the [Android SDK Manager](https://learn.microsoft.com/xamarin/android/get-started/installation/android-sdk) to install the necessary SDKs.
 
-### Opening the Repository
+### Building the Build Tasks
+Before opening the solution in Visual Studio you **MUST** build the build tasks:
 
 ```dotnetcli
 dotnet tool restore
-dotnet cake --target=VS --workloads=global
+dotnet cake --target=dotnet-buildtasks
```

Comment:
We should add this as an option or additional path or for information purposes but not as the primary path.

Reply:
Why not the primary path? I can add more info saying you can also use cake to launch VS, but what should I give in the doc as the advantage of that approach, why folks may want to use it?

Reply:
Is it just a convenient way to launch VS & open the solution from the command line? Or are there other advantages?

Reply:
Also, is the expectation that folks normally/always launch VS with `dotnet cake --target=VS --workloads=global` rather than launching VS directly, beyond just doing that the first time? If so, that seems a good deal slower (it takes an extra 70 seconds or so on my devbox, even when the build tasks are already built).

Reply:
It's safer to keep people silo'd into a script. It's hard to know what might change or break down the line. We can pivot the build steps underneath them if we need to. 

for example, VSMAC was broken with our sln up until a few months ago. Once it was fixed we just changed the cake script to launch a different SLN.

Reply:
@PureWeen - I updated, adding the cake and non-cake options and tweaked the text/formatting a bit as a consequence. I think it's pretty good, but please take a look to review. Thanks much.